csv_exploration_lesson/readme.md

```diff
@@ -12,6 +12,10 @@ I started these notes after writing down to [A Digression to Turing's "CSV Explo
 
 ----------------------
 
+## Notes/gotchas from others who have worked through this guide 
+
+- [Jason Gong's notes on the lesson] (https://gist.github.com/mhpjay422/4fdfb650910b18e82ab7d1731181edf3)
+
```

Comment:
this is perfect! Could you add a line break after this section? 

A `-----------` after your entry.

(As others work through this in the future, I want them to add _their_ notes, and have their notes delineated from the rest of the doc)